embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/features/ResurrectionFeatureTest.kt

```diff
@@ -0,0 +1,52 @@
+package io.embrace.android.embracesdk.testcases.features
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.embrace.android.embracesdk.fakes.behavior.FakeAutoDataCaptureBehavior
+import io.embrace.android.embracesdk.testframework.IntegrationTestRule
+import org.junit.Assert.assertNotNull
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class ResurrectionFeatureTest {
```

Comment:
I'd expect a test case that validates the service is capable of resurrecting payloads, but that could come in a different PR

Reply:
Yeah, this thought I needed to fake the file system before I can do this, but if I don't, I'll add it here before I merge this

Reply:
I'm going to need the changes to the integration tests that changes the assertions to verify using the v2 delivery layer before I can write the tests. I noticed that I wasn't getting sessions back and realized that we still need to merge part of [this](https://github.com/embrace-io/embrace-android-sdk/pull/1472), specifically the EmbraceAssertionInterface changes.